index.js

```diff
@@ -175,6 +187,28 @@ const parseArgs = (
   return result;
 };
 
+// Look ahead to next token in argv array:
+function peek(argv, pos) {
+  if (pos + 1 >= argv.length) {
+    return 0;
+  }
+  return token(argv[pos + 1]);
+}
+
+// Guess token type based on individual string from argv:
+function token(arg) {
+  const chr = StringPrototypeCharAt(arg, 0);
+  const nextChr = StringPrototypeCharAt(arg, 1);
+  if (StringPrototypeMatch(chr, /^[0-9]$/)) {
+    return TOKENS.NUMERIC;
+  } else if (chr === '-' && StringPrototypeMatch(nextChr, /^[0-9]$/)) {
```

Comment:
- Can drop `^` and `$` since were only matching one character (Unless I'm overlooking something).
- Should we use `\d` for increased unicode support, or does that add more complexity/edge cases?
- Should we be using `.test()` since were not using the result of `.match()`
  e.g. `/[0-9]/.test(chr)`

Reply:
Yes, good call, this should use RegExpPrototypeTest

Reply:
I needed to double check, but I believe `\d` is the same as `[0-9]` so a matter of taste.

Reply:
> I needed to double check, but I believe `\d` is the same as `[0-9]` so a matter of taste.

Needed to double checked myself! Got my languages mixed up (: https://regex101.com/r/PhKpmF/1